app/controllers/claims_controller.rb

```diff
@@ -72,6 +73,10 @@ def check_page_is_in_sequence
     raise ActionController::RoutingError.new("Not Found") unless page_sequence.in_sequence?(params[:slug])
   end
 
+  def check_claim_not_in_progress
+    redirect_to(existing_session_path(policy: params[:policy])) && return if session[:claim_id].present?
```

Comment:
Yeah, the `&& return` was me misremembering about using `redirect_to` in a conditional. I don't think it needs to be there if there's no other logic in the method

Reply:
I think it would make sense for the title to match the user's intent when they hit the warning, which would be the policy in the params (ie url and title match, error message tells you what the problem is)

Reply:
> I think it would make sense for the title to match the user's intent when they hit the warning, which would be the policy in the params (ie url and title match, error message tells you what the problem is)

Yeah, I think this would be the best.